### ISSUE TITLE
Using runOnly with tags for wcag21 results in passing tests with 0 violations

### ISSUE THREAD

**Issue by tzbarkan** [component-driven/cypress-axe#123]
If I use runOnly with tags for `wcag2a` and `wcag2aa`, I find violations as expected:
`cy.checkA11y(subject, { runOnly: { type: "tag", values: ["wcag2a", "wcag2aa"] } })`

But if I use the tags `wcag21a` and `wcag21aa` as documented in axe-core 4.4.1, no violations are found:
`cy.checkA11y(subject, { runOnly: { type: "tag", values: ["wcag21a", "wcag21aa"] } })`

Expectation: using `wcag21a` and `wcag21aa` should return a comparable number of violations. Most likely more.

**Comment by AndreaPontrandolfo** [comment]
> If I use runOnly with tags for `wcag2a` and `wcag2aa`, I find violations as expected: `cy.checkA11y(subject, { runOnly: { type: "tag", values: ["wcag2a", "wcag2aa"] } })`
> 
> But if I use the tags `wcag21a` and `wcag21aa` as documented in axe-core 4.4.1, no violations are found: `cy.checkA11y(subject, { runOnly: { type: "tag", values: ["wcag21a", "wcag21aa"] } })`
> 
> Expectation: using `wcag21a` and `wcag21aa` should return a comparable number of violations. Most likely more.
> 
> Cypress 9.5.3 Cypress Axe 0.14.0 axe-core 4.4.1 with cypress-watch-and-reload 1.7.4
> 
> on Windows 10

I'm having the exact same issue.

Cypress 9.7.0
Cypress Axe 0.14.0
axe-core ^4.4.2
Windows 10

**Comment by bjornlauwerijs** [comment]
I am having the same issue. I expect a higher level to have more issues, but I get none on all tags with wcag**21**a, wcag**21**aa, wcag**21**aaa. I also got some strange results with running wcag**2**a compared to wcag**2**aa and wcag**2**aaa.

Screenshot below explains it better (test is always running on the same page but with different tags).

![2022-07-13 10_02_53-IBSBELGIUM_REPORT-screenshot](https://user-images.githubusercontent.com/87498356/179479030-ba1e25a3-63bd-48a6-8c8f-4203f49f9f19.png)

**Comment by nurulquamar** [comment]
I am also facing the same issue. wcag21a and wcag21aa are not finding any violations. But when I use wcag2a and wcag2aa I am able to. Also, these violations clearly mention that they are part of wcag21a as well.

axe-core: 4.4.3
cypress-axe: 1.0.0
cypress": 9.5.2
Mac OS 12.6

**Comment by nurulquamar** [comment]
I opened an issue with axe-core and it was closed with a comment. See the below link if it helps:
https://github.com/dequelabs/axe-core/issues/3760

**Comment by vladttt1123** [comment]
can someone tell me what is the befir of using : 
values: ['wcag2a', 'wcag2aa'],
If as per the documentation that I found: 
==
The WCAG criteria are cumulative—to conform to WCAG Level AA, you must first conform to A. To conform to WCAG Level AAA accessibility criteria, you just first conform to A and AA.
==

So using only  wcag2aa is sufficient, or am I missing something ?

**Comment by gquinteros93** [comment]
Based on this commment:
- https://github.com/Enterprise-CMCS/managed-care-review/pull/2715#discussion_r1739319694 

Looks like that by adding `['wcag21a', 'wcag21aa'],`  then only the specific rules added in for that standard will be tested.

Because of that, in order to highlight all the error we also need to include  `['wcag2a', 'wcag2aa'],` as follows:

`cy.checkA11y(subject, { runOnly: { type: "tag", values: ['wcag2a', 'wcag2aa', 'wcag21a', 'wcag21aa'] } })`
